document_test.go

```diff
@@ -240,21 +275,18 @@ func TestDocument_RenderAndReload_ChangeCheck_Asana(t *testing.T) {
 
 	assert.Nil(t, errs)
 	tc := compReport.TotalChanges()
-	assert.Equal(t, 21, tc)
```

Comment:
is this accurate?

Reply:
https://github.com/pb33f/libopenapi/pull/205#discussion_r1426586957

Just to clarify I believe this is accurate but at the same time a lot of these magic numbers were very hard to validate, what would have been better were more specific tests where you could have been asserting actually changes instead of just counts of changes